app/src/test/kotlin/compiler/intermediate_form/FunctionDependenciesAnalyzerTest.kt

```diff
@@ -116,6 +117,106 @@ class FunctionDependenciesAnalyzerTest {
         assertNotEquals("globals", globalsIdentifier.value)
     }
 
+    @Test fun `test functions created in nested block receive identifiers`() {
+        /*
+         czynność some_prefix() {
+             jeżeli (prawda) {
+                 czynność funkcja1() {}
+             }
+             dopóki (prawda) {
+                 czynność funkcja2() {}
+             }
+             {
+                 czynność funkcja() {}
+             }
+         }
+         */
+        val function = Function("function", listOf(), Type.Unit, listOf())
```

Comment:
To actually resemble the program this should be
```suggestion
        val function = Function("funkcja1", listOf(), Type.Unit, listOf())
```
(and so on)